examples/tdec/compound_multichain_tdec.py

```diff
@@ -0,0 +1,112 @@
+import os
+
+from nucypher_core.ferveo import DkgPublicKey
+
+from nucypher.blockchain.eth.agents import CoordinatorAgent
+from nucypher.blockchain.eth.registry import InMemoryContractRegistry
+from nucypher.characters.lawful import Bob, Enrico
+from nucypher.policy.conditions.lingo import ConditionLingo
+from nucypher.utilities.logging import GlobalLoggerSettings
+
+######################
+# Boring setup stuff #
+######################
+
+LOG_LEVEL = "info"
+GlobalLoggerSettings.set_log_level(log_level_name=LOG_LEVEL)
+GlobalLoggerSettings.start_console_logging()
+
+staking_provider_uri = os.environ["DEMO_L1_PROVIDER_URI"]
+network = "lynx"
+
+coordinator_provider_uri = os.environ["DEMO_L2_PROVIDER_URI"]
+coordinator_network = "mumbai"
+
+###############
+# Enrico
+###############
+
+print("--------- Threshold Encryption ---------")
+
+coordinator_agent = CoordinatorAgent(
+    provider_uri=coordinator_provider_uri,
+    registry=InMemoryContractRegistry.from_latest_publication(
+        network=coordinator_network
+    ),
+)
+ritual_id = 91  # got this from a side channel
```

Comment:
Just a nitpick: Is there any reason why `DEMO_{L1,L2}_PROVIDER_URI` are env variables, and `network`, `coordinator_network` and `ritual_id` aren't?